reindexer/complete_reindex/src/complete_reindex.py

```diff
@@ -4,10 +4,23 @@
 
 from boto3.dynamodb.conditions import Attr
 import boto3
-
+from botocore.exceptions import ClientError
+from tenacity import *
 from wellcome_aws_utils import sns_utils
 
 
+def should_retry(err):
+    if isinstance(err, ClientError):
+        if err.response['Error']['Code'] == 'ProvisionedThroughputExceededException':
```

Comment:
minor: you could simplify this even further:

```python
return err.response['Error']['Code'] == 'ProvisionedThroughputExceededException'
```

Reply:
Hmm, what if it's not a ClientError?

Reply:
Then you won't enter this branch.

To be clear, I meant 14–15, so you’d have

```python
if isinstance(err, ClientError):
    return err.response['Error']['Code'] == 'ProvisionedThroughputExceededException'

return False
```

Reply:
Ah yeah. Good.